csrc/python_frontend/fusion_definition.cpp

```diff
@@ -187,8 +212,10 @@ std::vector<at::Tensor> FusionDefinition::execute(
     }
   }
 
-  outputs = scheds->auto_gen_schedules->runFusionWithInputs(
-      inputs, std::nullopt, selected_device);
+  if (outputs.empty()) {
```

Comment:
Why is this check needed?

Reply:
I think I fixed the code. sorry for the ugly patch. I should have done a better job.

If you look at line 200-213. When `!override_user_schedule` is true, we shouldn't run automatic scheduled fusion. I'll clean it up. :bow: